ksl_notify.py

```diff
@@ -133,43 +152,57 @@ def main(args):
         logging.getLogger().setLevel(numeric_level)
 
     # Get needed controls
-    loop_delay = args.pop('time') * 60
-    email = args.pop('email', None)
+    minutes = args.pop('time')
+    loop_delay = minutes * 60
+    sender = args.pop('email', None)
```

Comment:
Please test the `load` parameter if present and valid before prompting for email. If the user has messed up the parameter, it's best to report the problem before asking for input.

Reply:
Done.